#ip_address: "192.168.178.59" # if set, rosbridge will use WiFi insted of USB

#from swiftrobotm to ros
input:
  - ros_topic: /ultrawideangle_camera/image_raw
    ros_type: sensor_msgs/Image
    swift_channel: 1

  
# from ros to swiftrobotm
